lib/features/pokemon_overview/widgets/pokemon_card.dart

```diff
@@ -18,7 +18,7 @@ class PokemonCard extends StatelessWidget {
       child: Column(
         mainAxisAlignment: MainAxisAlignment.center,
         children: [
-          Image.network(imageUrl.replaceAll(indexKey, pokemon.url.getPokemonId)),
+          Expanded(child: Image.network(imageUrl.replaceAll(indexKey, pokemon.url.getPokemonId))),
```

Comment:
wrap the imageUrl in a new variable inside the build method and have it readily available prior to its return statement to make this more readable 🙂